problem_builder/public/js/instructor_tool.js

```diff
@@ -253,8 +253,9 @@ function InstructorToolBlock(runtime, element) {
     // Block types with answers we can export
     var questionBlockTypes = ['pb-mcq', 'pb-rating', 'pb-answer'];
 
+    // Fetch this course's blocks from the REST API, and adds the to the
+    // list of blocks in the Section/Question drop-down list.
```

Comment:
@pomegranited Typos: "Fetch this course's blocks ..., and **add** the ??? to the list of blocks ..." (perhaps you meant to say "and add **them**"?)